modules/virt-removing-custom-mcp.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_networking/virt-using-dpdk-with-sriov.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-removing-custom-mcp_{context}"]
+= Removing a custom machine config pool for high-availability clusters
+
+You can delete a custom machine config pool that you previously created for your high-availability cluster.
+
+.Prerequisites
+* You have access to the cluster as a user with `cluster-admin` permissions.
+* You have installed the OpenShift CLI (`oc`).
+* You have created a custom machine config pool by labeling a subset of the compute nodes with a custom role and creating a `MachineConfigPool` manifest with that label.
+
+.Procedure
+
+. Remove the `worker-dpdk` label from the compute nodes by running the following command:
++
+[source,terminal]
+----
+$ oc label node <node_name> node-role.kubernetes.io/worker-dpdk-
+----
+
+. Delete the `MachineConfigPool` manifest that contains the `worker-dpdk` label by entering the following command:
```

Comment:
```suggestion
. Delete the `MachineConfigPool` manifest that contains the `worker-dpdk` label by running the following command:
```
Just for consistency.

Reply:
Thanks for the suggestion. I would like to keep it as is because both are acceptable per the contribution guide.